astropy/io/votable/tests/data/parquet_binary.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Produced with astropy.io.votable version 6.0.dev716+g6133084e35
```

Comment:
If the commit hash is from this PR, not sure how useful it is for future debugging. Maybe better to link to this PR URL instead.

Reply:
OK, I can hack on it, but then the statement that "Produced with..." won't be truthful.